reviews/swirl-protocol/reentrancy/README.md

```diff
@@ -0,0 +1,72 @@
+# documentation
+## By: [Kevin Keaveney](https://github.com/kkeaveney)
+
+## Summary / Recommendations
```

Comment:
Expand this section with overall outlook of why this contract is reentrancy free. SafeERC + Access Control + not-a-lot-of-places-contract-does-external-calls.